jetbrains-core/src/software/aws/toolkits/jetbrains/services/telemetry/TelemetryService.kt

```diff
@@ -65,6 +65,15 @@ interface TelemetryService : Disposable {
             ApplicationManager.getApplication().messageBus.connect().subscribe(TELEMETRY_TOPIC, notifier)
         }
 
+        @JvmStatic
+        fun recordSimpleTelemetry(project: Project?, name: String, result: TelemetryConstants.TelemetryResult, count: Double = 1.0) =
```

Comment:
I like the `recordSimpleTelemetry` method. We seem to use a ternary type style a few times, does it make sense to have an overload that takes a `success` `Boolean` (or function that returns a `Boolean` as well?

Reply:
Sounds like something we can used in a few places, added